src/view/droppable-dimension-publisher/get-dimension.js

```diff
@@ -117,7 +117,7 @@ export default ({
 
     return {
       client: frameClient,
-      page: withScroll(frameClient),
```

Comment:
`withScroll` will apply the page scroll

Reply:
You are right that your change is a bit stricter in that it does not read the page scroll directly, but just supplies it..

Thoughts?

Reply:
I also like that this approach does not touch the DOM again and just uses the value that has been passed in